src/textcode/strings.py

```diff
@@ -206,7 +206,7 @@ def is_relative_path(s):
 def is_win_path(s):
     """
     Return True if s looks like a win path.
-    Example: c:\usr\lib\librt.so.1.
+    Example: c:\\usr\\lib\\librt.so.1.
```

Comment:
Repetition of https://github.com/nexB/scancode-toolkit/pull/1534/commits/1584e60892c7f2cf7564f7129179abeea84e39d8